.github/workflows/test.yml

```diff
@@ -40,7 +40,12 @@ jobs:
 
       - name: Test core with pytest
         run: |
-          pytest tests -vv --skip-ocpapi-integration --cov-report=xml --cov=fairchem -c ./packages/fairchem-core/pyproject.toml
+          pytest tests -vv --ignore=tests/demo/ocpapi/tests/integration/  --cov-report=xml --cov=fairchem -c ./packages/fairchem-core/pyproject.toml
+
+      - name: Integration tests
```

Comment:
Will this run on every PR / commit on a PR? Or does it need to be targeted in some way?

These tests run the end to end demo so can get expensive if done on a CI workflow with lots of executions.

Reply:
so the by default this was running on every PR, if we want we can remove from CI all together, wdyt?

Reply:
Sorry, still catching up on how these tests are triggered. I assumed `--skip-ocpapi-integration` was skipping everything, but I think I understand now...

I would recommend adding `--skip-ocpapi-integration` to the command below. It was added in https://github.com/FAIR-Chem/fairchem/commit/2715160d92f833fda1454e29ade8095256b3d0f2 in order to skip some especially expensive API calls (ones that are actually behind a pretty restrictive rate limiter as well).

In addition, I'm not sure the right test methods are decorated. These methods _should_ have the decorator - they are very expensive and rate limited (I would not change them):
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/client/test_client.py#L169-L170
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/client/test_client.py#L261-L262

However, these methods are _not_ very expensive. I'm not sure why they have the decorator (@misko ?). I would recommend removing it from:
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/client/test_client.py#L353
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/client/test_client.py#L366
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/client/test_client.py#L393

Finally, this test does hit an expensive / rate-limited API. I would recommend adding the decorator to it:
- https://github.com/FAIR-Chem/fairchem/blob/main/tests/demo/ocpapi/tests/integration/workflows/test_adsorbates.py#L73

Reply:
One question on adding `continue-on-error: true`...

What happens if someone changes the client code in this repo in a way that actually break it? I understand the motivation here - that we don't want to block builds because a remote API is not behaving correctly. But this will hide breaking code changes as well, won't it?

Reply:
Moved integration tests out and re-enabled the above tests, removed the continue-on-error